Source/WebKit/Shared/WebCoreArgumentCoders.serialization.in

```diff
@@ -2358,6 +2358,29 @@ class WebCore::PasteboardCustomData {
     Vector<WebCore::PasteboardCustomData::Entry> data();
 };
 
+#if USE(UNIX_DOMAIN_SOCKETS)
```

Comment:
Moving this here from its own file is arguably a step in the wrong direction, but not a big deal.